src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx

```diff
@@ -109,7 +110,7 @@ export default function SwapModalHeader({
               {slippageAdjustedAmounts[Field.OUTPUT]?.toSignificant(6)} {trade.outputAmount.currency.symbol}
             </b>
             {/* {' or the transaction will revert.'} */}
-            {' or the transaction will expire.'}
+            {'or the swap will not execute. ' + INPUT_OUTPUT_EXPLANATION}
```

Comment:
Keep the space

```suggestion
            {' or the swap will not execute. ' + INPUT_OUTPUT_EXPLANATION}
```

Reply:
![screenshot_2021-03-08_07-26-37](https://user-images.githubusercontent.com/43217/110341829-9fc73800-7fdf-11eb-8540-9b76de4d4ed3.png)